.travis.yml

```diff
@@ -40,6 +40,19 @@ install:
     - cd test-server && pip install -r requirements.txt && cd ..
     - ./node_modules/.bin/webdriver-manager update
 
+before_script: >
+    df -h
```

Comment:
Please leave a comment here explaining what this bit does. It will be very helpful for others in the future. While it may be obvious to a dev familiar with the implementation, it is less obvious to a front-end dev.